rose/client/car.py

```diff
@@ -1,5 +1,10 @@
+import logging
+import random
+import os
+from rose.common import config
```

Comment:
These imports were just removed on master and are not needed after #219 and #227.